resources/shaders/simple_blur.fs

```diff
@@ -0,0 +1,29 @@
+#version 450
+
+layout(location = 0) out vec4 outBlur;
+
+layout(binding = 0) uniform sampler2DMS uInput;
+
+const int numSamples = 4;
+
+void main() {
+	ivec2 pos 		= ivec2(gl_FragCoord.xy);
```

Comment:
Only use tabs for indentation. The tabs after ivec2 pos and vec4 f_blur result in different location of the "=" sign.